packages/management/tkr-service/bundle/config/overlays/overlay-tkr-service.yaml

```diff
@@ -0,0 +1,80 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#@ if data.values.tkrService.createNamespace:
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: #@ data.values.tkrService.namespace
+#@ end
```

Comment:
We don't need this if we treat the namespace as a pre-req.

Reply:
Updated